testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -111,6 +113,20 @@ impl EventObserver {
         }
     }
 
+    fn make_new_microblocks_payload(receipts: Vec<StacksTransactionReceipt>) -> serde_json::Value {
+        let mut tx_index = 0;
+        let mut serialized_txs = Vec::new();
+        for receipt in receipts.iter() {
+            let payload = EventObserver::make_new_block_txs_payload(receipt, tx_index);
+            serialized_txs.push(payload);
+            tx_index += 1;
```

Comment:
Is this ordering deterministic? As in would all running stacks-nodes report microblock transactions with the same `tx_index`?

Reply:
Yes -- the transaction receipts from the unconfirmed microblock processing are generated in the order that they will also be (potentially) confirmed.  However, because the microblocks aren't confirmed yet, it's possible that you'll get receipts for transactions that will ultimately get dropped (e.g. because the next anchored block miner doesn't have to confirm them).